event_operating_unit/tests/test_operating_unit_user_id.py

```diff
@@ -50,38 +50,11 @@ def setUpClass(cls):
             'date_end': '2018-03-01 11:00:00'
         })
 
-    def test_compute(self):
-        users = self.event1.operating_unit_user_ids
-        # The demo user is in all ous so we have 3 instead of 2
-        self.assertEqual(len(users), 3)
-        # We should find user1 which is only in ou 1 and user3
-        # which is in both.
-        self.assertIn(self.user1, users)
-        self.assertIn(self.user3, users)
-        # But user2 doesn't belong to our orgunit
-        self.assertNotIn(self.user2, users)
+    def test_permission(self):
```

Comment:
IMO the tests should be broaden to cover the four CRUD operations when defining new record rules.